src/main/java/com/github/piedpiper/node/stepfunctions/StepFunctionsServiceNode.java

```diff
@@ -0,0 +1,45 @@
+package com.github.piedpiper.node.stepfunctions;
+
+import java.util.Optional;
+import java.util.function.Function;
+
+import com.github.piedpiper.common.PiedPiperConstants;
+import com.github.piedpiper.node.BaseNode;
+import com.github.piedpiper.node.NodeInput;
+import com.github.piedpiper.node.NodeOutput;
+import com.github.piedpiper.node.ParameterMetadata;
+import com.google.common.collect.Lists;
+
+public class StepFunctionsServiceNode extends BaseNode {
+
+	public static final ParameterMetadata METHOD = new ParameterMetadata("method", ParameterMetadata.MANDATORY,
+			Lists.newArrayList("EXECUTE"));
```

Comment:
Please add more methods tested so that you know this extends well and u have not made the functionality function specific. Atleast 2 methods